src/api/app/models/cloud/upload_job.rb

```diff
@@ -39,6 +41,16 @@ def target_validator_class
       @target_validator_class ||= "::Cloud::Params::#{target.capitalize}".constantize
     end
 
+    def to_xml(options = {})
```

Comment:
Why is this necessary? I'm missing where it is used?

Reply:
And this is used when rendering the upload jobs in [the controller](https://github.com/openSUSE/open-build-service/pull/4691/files#diff-4016cd1c604738f10a2f27ccf98bea94R34)